src/map/script.hpp

```diff
@@ -2210,6 +2210,20 @@ enum e_eleminfo : uint8 {
 	ELEMINFO_CLASS,
 };
 
+/* getguildinfo script command */
+enum e_guildinfo : uint8 {
+	GUILDINFO_NAME = 0,
```

Comment:
Should we start the enum from 1 to ensure the parameter (in the npc script) is defined correctly?
```suggestion
	GUILDINFO_NAME = 1,
```

Reply:
Sounds good to me.